app/src/main/java/com/sopt/dive/MainActivity.kt

```diff
@@ -1,174 +1,143 @@
-import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
-import androidx.compose.material3.Button
-import androidx.compose.material3.Scaffold
+import androidx.compose.foundation.layout.width
 import androidx.compose.material3.Text
-import androidx.compose.material3.TextField
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.platform.LocalContext
-import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.sopt.dive.ui.theme.DiveTheme
-import kotlin.jvm.java
+import androidx.compose.ui.Alignment
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        enableEdgeToEdge()
+        val id = intent.getStringExtra("id") ?: ""
+        val pw = intent.getStringExtra("pw") ?: ""
+        val nickname = intent.getStringExtra("nickname") ?: ""
+        val mbti = intent.getStringExtra("mbti") ?: ""
+
         setContent {
             DiveTheme {
-                SignInScreen()
+                MainScreen(id = id, pw = pw, nickname = nickname, mbti = mbti)
             }
         }
     }
 }
 
 @Composable
-fun Boxes() {
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .background(Color.White),
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.SpaceEvenly
-    ) {
-        Box(
-            modifier = Modifier
-                .size(150.dp)
-                .background(Color.Red)
-        )
-        Box(
-            modifier = Modifier
-                .size(150.dp)
-                .background(Color.Green)
-        )
-        Box(
-            modifier = Modifier
-                .size(150.dp)
-                .background(Color.Blue)
-        )
-
-    }
-}
-
-@Composable
-fun SignInScreen() {
-    var id by remember { mutableStateOf("") }
-    var pw by remember { mutableStateOf("") }
-    val context = LocalContext.current
-
+fun MainScreen(id: String, pw: String, nickname: String, mbti: String) {
```

Comment:
```suggestion
fun MainScreen(
    id: String,
    pw: String,
    nickname: String,
    mbti: String
) {
```

여러개의 파라미터들을 작성하게 될때 저는 이렇게 줄바꿈 하는게 더 읽기 편하더라구요 !!